awscli/s3transfer/crt.py

```diff
@@ -206,6 +206,7 @@ def upload(self, fileobj, bucket, key, extra_args=None, subscribers=None):
             extra_args = {}
         if subscribers is None:
             subscribers = {}
+        self._validate_checksum_algorithm_supported(extra_args)
```

Comment:
somewhat offtopic: Noticing that CRTTransferManager doesn't validate input as extensively as the classic TransferManager:
https://github.com/aws/aws-cli/blob/d61d07799c5c4d2d40d5e26c6d6efed474f98501/awscli/s3transfer/manager.py#L318-L319

Reply:
Yeah I think we should follow up on validating extra args in another PR.